CHANGELOG.md

```diff
@@ -90,8 +77,25 @@ for specific instructions.
   also been deprecated in favor of the same flags with the `metrics.` prefix.
   (@rfratto)
 
-- [ENHANCEMENT] `agentctl cloud-config` now allows specifying an url for
```

Comment:
Heads up @cristiangreco, I removed this since we don't currently publicly document the `agentctl cloud-config` command.

Reply:
No worries, I followed the PR guidelines but wasn't really sure whether it was something we want to mention.